ARrehab/ARrehab/Minigames/FaceGame/FaceViewController.swift

```diff
@@ -0,0 +1,358 @@
+//
+//  ARFaceViewController.swift
+//  ARrehab
+//
+//  Created by Alan Zhang on 2020/6/6.
+//  Copyright © 2020 Eric Wang. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import ARKit
+
+
+/**
+ Face Minigame ViewController
+ */
+class FaceViewController: UIViewController, ARSessionDelegate {
+    
+    //changing elements
+    @IBOutlet weak var actionLabel: UILabel!
+    @IBOutlet weak var timeLeftBar: UIProgressView!
+    @IBOutlet weak var pointsLeftLabel: UILabel!
+    @IBOutlet weak var totalPointsLabel: UILabel!
+    @IBOutlet weak var livesLeftLabel: UILabel!
+    
+    //static elements
+    @IBOutlet weak var livesLeftTitleLabel: UILabel!
+    
+    //action buttons
+    @IBOutlet weak var startGameButton: UIButton!
+    
+    //views
+    @IBOutlet weak var sceneView: SCNView!
+    @IBOutlet weak var faceImageView: UIImageView!
+    
+    let session = ARSession()
+    var maskNode: Mask?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.sceneView.backgroundColor = .clear
+        self.sceneView.scene = SCNScene()
```

Comment:
we are using RealityKit now not SceneKit

Reply:
Also consider using the scene already generated by the main view controller and just changing the configuration.